sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -17,6 +17,37 @@ Use the client library for App Configuration to create and manage application co
 - [App Configuration Store][app_config_store]
 
 ### Include the Package
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on the latest GA version of the library.
```

Comment:
I was thinking this could be more directive than suggestive.

```suggestion
Include `azure-sdk-bom` as a `dependencyManagement` dependency to allow it to manage Azure SDK dependencies in your project.
```